api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -103,6 +103,16 @@
         "column": educational_models.CollectiveOffer.venueId,
     },
     "VALIDATION": {"field": "validation", "column": educational_models.CollectiveOffer.validation},
+    "PRICE": {
+        "field": "price",
+        "column": aliased_stock.price,
+        "inner_join": "stock",
+        "custom_filter_all_operator": sa.and_(
```

Comment:
Essayons d'écrire sans fautes
```suggestion
        "custom_filter_all_operators": sa.and_(
```

Reply:
`"custom_filter_all_operators": sa.and_(`
modifié dans : 
collective_offer_blueprint ligne 110
blueprint d'offer ligne 172
utils ligne 271